pgbouncer/datadog_checks/pgbouncer/pgbouncer.py

```diff
@@ -206,6 +206,7 @@ def check(self, instance):
         password = instance.get('password', '')
         tags = instance.get('tags', [])
         database_url = instance.get('database_url')
+        use_cached = instance.get('use_cached', True)
```

Comment:
Can you please wrap this with `is_affirmative`? `from datadog_checks.config import is_affirmative`